tests/src/system/basic/WskActionSequenceTests.scala

```diff
@@ -125,7 +125,6 @@ class WskActionSequenceTests
 
             val stdout = wsk.action.get(name).stdout
             assert(stdout.startsWith(s"ok: got action $name\n"))
-            wsk.parseJsonString(stdout).fields("exec").asJsObject.fields("components") shouldBe compValue
```

Comment:
the names of the components are modified in the new version of seqs ('_' gets replaced by the user namespace). I'll look to see if I can modify the check to still have it in (with proper values).